### DISCUSSION TITLE
Public roadmap / backlog

### DISCUSSION THREAD

**Discussion by wiesson** [supabase/supabase#4889] [General]
Is there a public roadmap or public backlog so that I can see what features might come in the future?

**Comment by eqoram** [comment]
Any update on this? :)

**Comment by gruckion** [comment]
lol!?

**Comment by masaok** [comment]
Not exactly the future, but better than nothing: https://supabase.com/changelog

**Comment by marcospgp** [comment]
2025 and still no roadmap or way for users to vote on features they care about (other than perhaps reacting to github issues, although those seem to be overlooked 😭)

**Reply by omranjamal** [reply]
I mean yeah the roadmap isn't there, but feature requests and votes are public on this discussion board. 🤷
